ttkwidgets/font/familydropdown.py

```diff
@@ -49,7 +54,7 @@ def selection(self):
         :return: None if no font is selected and font family name if one is selected.
         :rtype: None or str
         """
-        if self._font.get() is "" or self._font.get() not in self._fonts:
+        if self._font.get() == "" or self._font.get() not in self._font_families:
```

Comment:
The first part of this condition is useless. An empty string won't be in the font family list anyways.